<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="FindButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Find!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditFunction">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>(120000-M)/C</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>481</width>
      <height>231</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1</width>
      <height>0</height>
     </size>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>110</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>21</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Make-Model</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Year</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Odometer</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cost</string>
     </property>
    </column>
   </widget>
   <widget class="QCheckBox" name="CbSpecialParameters">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Special Parameters</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>(M)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Search By:</string>
    </property>
    <widget class="QCheckBox" name="CBState">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblYearMax">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditYearMax">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxState">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>72</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>al</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ak</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>az</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ar</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ca</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>co</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ct</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dc</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>de</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>fl</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ga</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>hi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>id</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>il</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>in</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ia</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ks</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ky</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>la</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ks</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ky</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>la</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>me</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>md</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ma</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ms</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mt</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nv</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nh</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nj</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>mt</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nv</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nh</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ny</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nc</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>nd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>oh</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ok</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>or</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>pa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ri</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sc</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>sd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>tn</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>tx</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ut</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>vt</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>va</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>wa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>wv</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>wi</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>wy</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="EditCostMin">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>110</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblOdoMax">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditCostMax">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCostMax">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>110</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditOdoMax">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblYearMin">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>50</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Min:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditOdoMin">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditYearMin">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblOdoMin">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Min:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CBOdometer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Odometer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="EditMake">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lblCostMin">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>110</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Min:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CBMake">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Make</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CBYear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CBCost">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Cost</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemove">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>170</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>110</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>(C)</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Sort by:</string>
    </property>
    <widget class="QRadioButton" name="RBMin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="RBMax">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="lbltime1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Computational Time: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lbltime2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>630</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Computational Time: </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>791</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Sort Type:</string>
    </property>
    <widget class="QRadioButton" name="RBYear">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Year</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RBOdo">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Odo</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="RBCost">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>84</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Cost</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="RBFunction">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>141</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Special Parameters</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithem 1: Shell Sort</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>610</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithem 2 : Merge Sort</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTotalCarsSorted">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>280</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Cars Sorted through:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>481</width>
      <height>231</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1</width>
      <height>0</height>
     </size>
    </property>
    <property name="rowCount">
     <number>10</number>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>110</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>21</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Make-Model</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Year</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Odometer</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cost</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButtonPrev">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>560</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNext">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>560</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPage">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>560</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Page: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStates">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>180</y>
      <width>481</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>States: ALL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonNext_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>560</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPrev_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>560</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Prev</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPage_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>560</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Page: </string>
    </property>
   </widget>
   <widget class="QLabel" name="MM">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="C">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QLabel" name="Odo">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Odometer</string>
    </property>
   </widget>
   <widget class="QLabel" name="Y">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLabel" name="Y_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLabel" name="MM_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Make Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="Odo_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>300</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Odometer</string>
    </property>
   </widget>
   <widget class="QLabel" name="C_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>300</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPerfect_Car">
    <property name="title">
     <string>Perfect Car</string>
    </property>
   </widget>
   <addaction name="menuPerfect_Car"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
